src/functions.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+function displayDreamDetails(array $dreams): string{
+    $dreamOutput ='';
+    foreach(array_reverse($dreams) as $dream) {
+
+        if(array_key_exists('dream_title', $dream) 
+        && array_key_exists('dream_or_nightmare', $dream) 
+        && array_key_exists('dream_date', $dream) 
+        && array_key_exists('dream_description', $dream)) {
+
+            $dreamOutput .="<h3>" . $dream['dream_title'] . "</h3>"
+            . "<h4>" . $dream['dream_or_nightmare'] . "</h4>"
+            . "<h4>" . $dream['dream_date'] . "</h4>"
```

Comment:
h4 is not semantically used here, because the purpose of these elements is not to be headings, rather they are just text that looks slightly bigger than a p. i'd recommend using a different tag and styling them instead

Reply:
Change to p, and not much different in terms of look!

Reply:
Changed the styles too!